pkg/engine/secrets/inspector.go

```diff
@@ -612,15 +621,31 @@ func validateCustomSecretsQueriesID(allRegexQueries []RegexQuery) error {
 func (c *Inspector) checkContent(i, idx int, basePaths []string, files model.FileMetadatas) {
 	// lines ignore can have the lines from the resolved files
 	// since inspector secrets only looks to original data, the lines ignore should be replaced
+	numRoutines := 30
```

Comment:
change this to a constant

Reply:
modified the parallelism [here](https://github.com/Checkmarx/kics/pull/6858/commits/a353096e01ccbce4162f8d5f76ba3810b8bc0206)